pulp_smash/pulp3/utils.py

```diff
@@ -81,9 +81,44 @@ def sync(cfg, remote, repo, **kwargs):
         created sync.
     """
     client = api.Client(cfg)
-    data = {"repository": repo["pulp_href"]}
+    data = {"remote": remote["pulp_href"]}
     data.update(kwargs)
-    return client.post(urljoin(remote["pulp_href"], "sync/"), data)
+    return client.post(urljoin(repo["pulp_href"], "sync/"), data)
+
+
+def modify_repo(cfg, repo, base_version=None, add_units=None, remove_units=None):
```

Comment:
If `add_units is None`, and `remove_units is None,` but `base_version is not None,` it will attempt to modify a repository without units to be added or removed, is it an expected behave?  How pulp will handle this?

Should add logic to restrict this?

Reply:
I initially had that logic there, but I discovered that a bunch of tests do actually do exactly this (make a new version without changing anything at all).

Reply:
@dralley, Interesting.  I was expecting Pulp to raise an exception in case just `base_version` is provided.